sound/soc/codecs/rt1320-sdw.c

```diff
@@ -535,6 +535,9 @@ static int rt1320_read_prop(struct sdw_slave *slave)
 	/* set the timeout values */
 	prop->clk_stop_timeout = 64;
 
+	/* BIOS may set wake_capable. Make sure it is 0 as wake events are disabled. */
+	prop->wake_capable = 0;
```

Comment:
the `wake_capable_list[]` is _intel_ specific (drivers/soundwire/intel_auxdevice.c), the codec driver is not.
I'm not sure how other OSs can handle if the rt1320 is marked by BIOS as wake capable and why we cannot.

The rt1320 can be used on devices sporting other vendor's SoC..